ydb/core/tx/schemeshard/schemeshard_export__create.cpp

```diff
@@ -311,7 +327,24 @@ struct TSchemeShard::TExport::TTxProgress: public TSchemeShard::TXxport::TTxBase
             << ", txId# " << txId);
 
         Y_ABORT_UNLESS(item.WaitTxId == InvalidTxId);
-        Send(Self->SelfId(), BackupPropose(Self, txId, exportInfo, itemIdx));
+        if (item.SourcePathType == NKikimrSchemeOp::EPathTypeTable) {
+            Send(Self->SelfId(), BackupPropose(Self, txId, exportInfo, itemIdx));
+        } else if (item.SourcePathType == NKikimrSchemeOp::EPathTypeView) {
+            Ydb::Export::ExportToS3Settings exportSettings;
+            Y_ABORT_UNLESS(exportSettings.ParseFromString(exportInfo->Settings));
+            const auto databaseRoot = TStringBuilder() << '/' << JoinSeq('/', Self->RootPathElements);
```

Comment:
```suggestion
            const auto databaseRoot = CanonizePath(Self->RootPathElements);
```
Лучше использовать стандартные функции: так проще менять поведение (а вдруг), проще грепать, рефакторить и т.п.